router/core/operation_processor.go

```diff
@@ -569,6 +575,17 @@ func (o *OperationKit) normalizeNonPersistedOperation() (cached bool, err error)
 	return false, nil
 }
 
+func (o *OperationKit) nullifyExportedVariables(variables []byte) []byte {
+	if variables == nil {
+		return nil
+	}
+	// min meaningful variables = {"a":1}
+	if len(variables) < 7 {
+		return nil
```

Comment:
Why do we return nil when we have variables? Could you add more insights into the comment?

Reply:
added